.github/workflows/build.yml

```diff
@@ -94,8 +94,11 @@ jobs:
       - name: Build Engagements Data docker image
         shell: bash
         run: |
-          echo "ECR Docker build with tag ${NEW_VERSION}"
-          mvn clean package -Pretail-build -B docker:tag -Ddocker.image.tag=${NEW_VERSION} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+          echo "ECR Docker build retail with tag ${NEW_VERSION}"
+          mvn clean package -Pbuild-retail -B docker:tag -Ddocker.image.tag=${NEW_VERSION} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+
+          echo "ECR Docker build business with tag ${NEW_VERSION}"
+          mvn clean package -Pbuild-business -B docker:tag -Ddocker.image.tag=${NEW_VERSION} -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
Please also update the "Use maven-docker-plugin to build and push docker image to ECR" part with what we have on ModelBank